examples/axual/33_application_access_grant_authorizations.tf

```diff
@@ -0,0 +1,18 @@
+resource "axual_application_access_grant_authorization" "dash_consume_logs_example" {
+  application_access_grant = axual_application_access_grant.dash_consume_from_logs_in_example.id
+  status = "Approved" # Change this to Revoked in order to revoke approval. That can only be done after this current state has been applied
+}
```

Comment:
Asking the user to set the status to "Approved" seems strange since state of the grant is managed by the system and not the user directly. Have we considered representing the "state of approval" as a Terraform resource? For eg, the resource `axual_application_access_grant_approval` could refer to the `application_access_grant` inside without an explicit status field. The status field can remain internal and stored in Terraform state. In this case, another resource `axual_application_access_grant_rejection` will be required to reject the grant.

When data owner wants to revoke an existing approved grant, they can simply remove the `axual_application_access_grant_approval` resource.

Reply:
Makes sense. What of deleting the rejection? What would be the result?